packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -141,10 +141,12 @@ export const themeSchema = z
   })
   .strict()
 
-const publicThemeSchema = themeSchema.deepPartial().extend({
-  // to have `locale` and `text` as required properties
-  i18n: i18nSchema
-})
+const publicThemeSchema = themeSchema
+  .extend({
+    // to have `locale` and `text` as required properties
+    i18n: i18nSchema
+  })
+  .deepPartial()
```

Comment:
previously if `i18n` was set it requires `locale` and `text` to be set, now both are optional
✅ valid
```js
i18n: [{ locale: "en-US", text: "English" }]
```

❌ invalid
```js
i18n: [{ locale: "en-US" }]
```

Reply:
Ohh! I was running into an error without this change for docs/, it seems that the `i18n` field is required

Reply:
Ah, will take a look today at why this happened

Reply:
here `.optional()` was missing - `i18n: i18nSchema.optional()`